validphys2/src/validphys/process_options.py

```diff
@@ -37,7 +37,7 @@ class _Vars:
     abs_eta_2 = "abs_eta_2"
     eta_1 = "eta_1"
     eta_2 = "eta_2"
-    m_ll2 = "m_ll2"
+    M_ll = "M_ll"
```

Comment:
```suggestion
    M_ll = "M_ll"
    m_ll2 = "m_ll2"
```


During the merge you removed the variable `m_ll2` that some other process was already using.

Also, by convention we are using lowercase `m` for the invariant mass. This needs to be fixed also in the `metadata.yaml` and `kinematics.yaml`

Reply:
Ok, I think I was having issues with another file only accepting M_ll but I can try to fix this

Reply:
This should be fixed now